scripts/bootstrap

```diff
@@ -11,6 +11,8 @@ fi
 
 cd "$(dirname "$0")/../Examples"
 
-bundle config set path 'vendor/bundle'
+# uncomment the following if you get errors about not having access to the system's install
+# bundle config --local path 'vendor/bundle'
```

Comment:
It's actually pretty rare to mention this stuff in a script like this. If we want to guide people on their ruby setup, maybe we should do it in the readme instead, under a "Contributing" section?

Reply:
i'm happy to delete it. it's probably wrong-advice anyway.